### PR TITLE
Feature/aapd 13 db status removed

### PR DESCRIPTION
**Pull request by suityou01**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-13

## Description of change

Add remove status to constants

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [X] My commit history in this PR is linear
- [X] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/appeals-service-api/src/services/user.service.js [hunk 1]
```diff
@@ -183,7 +188,7 @@ const setUserStatus = async (id, status) => {
 			.collection('user')
 			.findOneAndUpdate(
 				{ _id: new ObjectId(id) },
-				{ $set: { updatedAt: new Date(), status: status } },
+				{ $set: { confirmedAt: new Date(), status: status } },
```
Comment (bryn-pins) [inline packages/appeals-service-api/src/services/user.service.js:191]: Would suggest confirmedAt is only set if status is confirmed, so it's not overwritten by other status changes

Something like:

```
let fieldsToSet = { status: status };

	if (status === STATUS_CONSTANTS.CONFIRMED) {
		fieldsToSet.confirmedAt = new Date();
	}
```

**Review by chrishawldev — DISMISSED** [review]
LGTM

File: packages/appeals-service-api/src/services/user.service.js (hunk 1 above)
Comment (bryn-pins) [reply packages/appeals-service-api/src/services/user.service.js:191]: Or can merge and do later, as I'm waiting on those statuses for one of my tickets

Comment (suityou01) [reply packages/appeals-service-api/src/services/user.service.js:191]: should there be other datetimestamps on this document like "removedAt"? Or is it better to stick with "updatedAt"?

Comment (bryn-pins) [reply packages/appeals-service-api/src/services/user.service.js:191]: Maybe just leave as is for now and can figure out in one of my PRs as I'm not sure
Only business requirement for these is to order by the confirmed at date

**Review by bryn-pins — APPROVED** [review]

**Review by bryn-pins — APPROVED** [review]

**Pull request merged**